python/Makefile

```diff
@@ -28,33 +23,33 @@ setup-dat: ## Download DAT test files
 .PHONY: build
 build: setup ## Build Python binding of delta-rs
 	$(info --- Build Python binding ---)
-	maturin build $(MATURIN_EXTRA_ARGS)
+	uvx --from 'maturin[zig]' maturin build $(MATURIN_EXTRA_ARGS)
 
 .PHONY: develop
 develop: setup ## Install Python binding of delta-rs
 	$(info --- Develop with Python binding ---)
-	maturin develop --extras=devel,pandas $(MATURIN_EXTRA_ARGS)
+	uvx --from 'maturin[zig]' maturin develop --extras=devel,pandas $(MATURIN_EXTRA_ARGS)
```

Comment:
Have you look at `uv build`, I'm not sure how mature that is though

Reply:
I've checked, it is not properly supported yet as a "standard" build backend. `uvx` allow to use the tool without trouble. `uv` uses everything that is officially supported by `python` these build backend are fully supported: hatchling, setuptools, Flit, PDM. I've seen work on maturin side to be more integrated with uv.
I am really not an expert on this topic, maybe there are better solutions.